de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/api/IPrintJobBL.java

```diff
@@ -53,10 +52,11 @@
 	 * This method is skipping items which were already printed (see {@link I_C_Printing_Queue#isProcessed()})
 	 * 
 	 * @param source
-	 * @param monitor
+	 * @param adPInstanceId
+	 * @param parentAsyncBatchId
 	 * @return number of created print jobs
 	 */
-	int createPrintJobs(IPrintingQueueSource source, IPrintJobMonitor monitor);
+	int createPrintJobs(IPrintingQueueSource source, int adPInstanceId, int parentAsyncBatchId);
```

Comment:
AFAIR we already talked about this and meanwhile i thought a bit and consulted @teosarca ..
now we saw that all the method does is to forward them to a printingasyncbatch in the finally block
teo made two suggestions:
* suggestion 1
  * let's extract those two ints into a POJO called e.g. `PrintJobContext`, and use that one as parameter
  * and let's also add a default method to this interface which is just `createPrintJobs(IPrintingQueueSource source) and which calls the other one one with an exsitng context.
  * goal: users that have no have no `pInstanceId` and no `parentAsyncBatchId` should feel no fud about calling a createPrintJobs method if this API
* suggestion two: somehow split this method. but with my current understanding, i can't even form an opinion about this